internal/operators/ocs/ocs_operator.go

```diff
@@ -2,19 +2,22 @@ package ocs
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/kelseyhightower/envconfig"
 	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/host/hostutil"
 	"github.com/openshift/assisted-service/internal/operators/api"
 	"github.com/openshift/assisted-service/internal/operators/lso"
 	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/conversions"
 	"github.com/sirupsen/logrus"
 )
 
 // operator is an OCS OLM operator plugin; it implements api.Operator
 type operator struct {
 	log    logrus.FieldLogger
-	config *Config
+	Config *Config
```

Comment:
@jakub-dzon , is it a good idea to make `Config` public? In CNV operator I don't see it public.

Reply:
I think it shouldn't be, especially that `operator` is private.